garrysmod/lua/includes/extensions/math.lua

```diff
@@ -67,14 +67,14 @@ math.Min = math.min
 	Name: EaseInOut(fProgress, fEaseIn, fEaseOut)
 	Desc: Provided by garry from the facewound source and converted
 			to Lua by me :p
-	Usage: math.EaseInOut(0.1, 0.5, 0.5) - all parameters shoule be between 0 and 1
+	Usage: math.EaseInOut(0.1, 0.5, 0.5) - all parameters should be between 0 and 1
 -----------------------------------------------------------]]
 function math.EaseInOut( fProgress, fEaseIn, fEaseOut )
 
 	if ( fEaseIn == nil ) then fEaseIn = 0 end
 	if ( fEaseOut == nil ) then fEaseOut = 1 end
 
-	if ( fProgress == 0 || fProgress == 1 ) then return fProgress end
+	if ( fProgress == 0 or fProgress == 1 ) then return fProgress end

```

Comment:
These changes are not in the domain of your PR.
Also GMod already uses Garry's operators for consistency.